addons/web/static/tests/views/fields/properties_field_tests.js

```diff
@@ -368,6 +379,13 @@ QUnit.module("Fields", (hooks) => {
             ".o_cp_action_menus span:contains(Add Properties)",
             "Show Add Properties btn in cog menu",
         );
+        const menuItems = target.querySelectorAll(".o_cp_action_menus span.o_menu_item")
+        for (const item of menuItems) {
+            if (item.textContent === "Add Properties") {
+                await click(item, null);
+                break;
```

Comment:
not a fan of this for loop => in master, could you just find the correct item with the appropriate selector (using :contains for instance) ?

Reply:
Me neither :( 
Okidoki, I will do it! 🙂